shop/src/main/java/ro/msg/learning/shop/utils/mapper/OrderMapper.java

```diff
@@ -0,0 +1,23 @@
+package ro.msg.learning.shop.utils.mapper;
+
+import ro.msg.learning.shop.dto.OrderDto;
+import ro.msg.learning.shop.model.Customer;
+import ro.msg.learning.shop.model.Location;
+import ro.msg.learning.shop.model.Order;
+
+import java.time.LocalDate;
+
+public class OrderMapper {
+    public static Order DtoToOrder(OrderDto orderDto) {
+        if (orderDto == null)
+            return null;
+        Location location = new Location(orderDto.getName(), orderDto.getCountry(), orderDto.getCity(),
```

Comment:
Same here, you can change the constructor to receive an orderDto and then use its attributes within it